.github/workflows/commitint.yml

```diff
@@ -0,0 +1,23 @@
+name: commitlint
```

Comment:
we can update CONTRIBUTING.md to say `is enforced in a commit hook and GitHub action via commitlint`